arroyo/backends/abstract.py

```diff
@@ -64,7 +64,7 @@ def unsubscribe(self) -> None:
     @abstractmethod
     def poll(
         self, timeout: Optional[float] = None
-    ) -> Optional[BrokerValue[TStrategyPayload]]:
+    ) -> Optional[BrokerValue[KafkaPayload]]:
```

Comment:
Also considered renaming this into just `Payload` and keeping `KafkaPayload` just as an alias